husky_bringup/package.xml

```diff
@@ -36,6 +36,7 @@
   <run_depend>tf2_ros</run_depend>
   <run_depend>um6</run_depend>
   <run_depend>ur_driver</run_depend>
+  <run_depend>ur_modern_driver</run_depend>
```

Comment:
How about the dependencies for PointGrey cameras, flir_ptu, velodyne and robotiq?